src/theme.js

```diff
@@ -1,122 +1,195 @@
+//imageHighLights- color for highlighting image in .svg
+//compImgHighlight- color for highlighting computer image in .svg (home)
+//jacketColor- jacket color in .svg
+//headerColor- header color for degree section in education
+//splashBg- splash text color
+
 export const blueTheme = {
   body: "#EDF9FE",
   text: "#001C55",
+  expTxtColor: "#000a12",
   highlight: "#A6E1FA",
   dark: "#00072D",
   secondaryText: "#7F8DAA",
   imageHighlight: "#0E6BA8",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#0A2472",
   headerColor: "#0E6BA877",
+  splashBg: "#001C55",
 };
 
 export const brownTheme = {
   body: "#FFFEFD",
   text: "#5D2A42",
+  expTxtColor: "#000a12",
   highlight: "#FFF9EC",
   dark: "#00072D",
   secondaryText: "#8D697A",
   imageHighlight: "#E29F95",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#FB6376",
   headerColor: "#E29F9577",
+  splashBg: "#5D2A42",
 };
 
 export const purpleTheme = {
   body: "#F8EFF4",
   text: "#231942",
+  expTxtColor: "#000a12",
   highlight: "#E0B1CB",
   dark: "#00072D",
   secondaryText: "#655E7A",
   imageHighlight: "#BE95C4",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#5E548E",
   headerColor: "#BE95C477",
+  splashBg: "#231942",
 };
 
-export const yelGreenTheme = {
+export const greenTheme = {
   body: "#FFFFEB",
   text: "#003F2F",
+  expTxtColor: "#000a12",
   highlight: "#dddf00",
   dark: "#00072D",
   secondaryText: "#4CA58F",
   imageHighlight: "#55a630",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#007f5f",
   headerColor: "#55a63077",
+  splashBg: "#003F2F",
 };
 
 export const redTheme = {
   body: "#FFF8E6",
   text: "#6a040f",
+  expTxtColor: "#000a12",
   highlight: "#ffba08",
   dark: "#03071e",
   secondaryText: "#964F56",
   imageHighlight: "#dc2f02",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#9d0208",
   headerColor: "#dc2f0277",
+  splashBg: "#6a040f",
 };
 
 export const blackTheme = {
   body: "#E5E5E5",
   text: "#14213d",
+  expTxtColor: "#000a12",
   highlight: "#ffffff",
   dark: "#000000",
   secondaryText: "#5A6377",
   imageHighlight: "#fca311",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#8d99ae",
   headerColor: "#fca31177",
+  splashBg: "#14213d",
 };
 
 export const pinkTheme = {
   body: "#FEE9F2",
   text: "#620E34",
+  expTxtColor: "#000a12",
   highlight: "#FBA7CD",
   dark: "#31071A",
   secondaryText: "#ef476f",
   imageHighlight: "#ef476f",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#8d99ae",
   headerColor: "#ef476f77",
+  splashBg: "#620E34",
 };
 
 export const violetTheme = {
   body: "#F4EEFC",
   text: "#430A58",
+  expTxtColor: "#000a12",
   highlight: "#D6BEF4",
   dark: "#21052C",
   secondaryText: "#875599",
   imageHighlight: "#9b5de5",
   compImgHighlight: "#E6E6E6",
   jacketColor: "#763D8B",
   headerColor: "#9b5de577",
+  splashBg: "#430A58",
 };
 
-export const greenTheme = {
+export const tealTheme = {
   body: "#E6FAF5",
   text: "#084c61",
+  expTxtColor: "#000a12",
```

Comment:
I would recommend you to undo all the changes other than adding material dark theme. We can do other changes in separate PR.